docs/guides/server/caching.adoc

```diff
@@ -159,8 +159,10 @@ In order to reduce memory usage, it's possible to place an upper bound on the nu
 cache. To specify an upper bound of on a cache, you must provide the following command line argument
 `--cache-embedded-$\{CACHE_NAME}-max-count=`, with `$\{CACHE_NAME}` replaced with the name of the cache you would like to
 apply the upper bound to. For example, to apply an upper-bound of `1000` to the `offlineSessions` cache you would configure
-`--cache-embedded-offline-sessions-max-count=1000`. An upper bound can not be defined on the following caches:
-`actionToken`, `authenticationSessions`, `loginFailures`, `work`.
+`--cache-embedded-offline-sessions-max-count=1000`. Conversely, to disable the upper-bound you would configure a negative value
+`--cache-embedded-offline-sessions-max-count=-1`.
```

Comment:
With @pruivo's recent changes that offline session even when they are not persistent use a bounded cache, I don't think that this is a good example any more. So those caches should always be set up in abounded manner. 

Thinking about it, I now can no longer see a cache that you would configure sometimes bounded, and sometimes unbounded. So maybe this is no longer needed?

Reply:
@ahus1 With small deployments, like we do in our weekly runs, it may make sense to have unbounded caches as everything fits in memory. It saves the cost of keeping track of the used entries for the eviction algorithm. 
If you don't want to explicitly document it, it is fine by me.

Reply:
I have updated the docs to use the `users` cache as an example instead, I have also tried to update the docs to reflect the rules currently enforced by `DefaultCacheEmbeddedConfigProviderFactory`.